server/Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'mongoid', '~> 4.0.0'
 gem 'symmetric-encryption'
 gem 'activesupport', '~> 4.1.14'
 gem 'faye-websocket', '~> 0.10.0'
+gem 'eventmachine', '>= 0.12.0'
```

Comment:
`CloudWebsocketConnectJob` and `Cloud::WebSocketClient` are using it. Should we get rid of it or just rely on `faye-websocket` will include it?